cocos/physics/framework/assets/physics-material.ts

```diff
@@ -61,15 +61,13 @@ export class PhysicsMaterial extends Asset {
      */
     @editable
     @type(CCFloat)
-    get friction (): number {
+    get friction(): number {
         return this._friction;
     }
 
-    set friction (value) {
-        if (!math.equals(this._friction, value)) {
```

Comment:
Perhaps, we need to check whether the value is changed by `math.approx` rather than `math.equals`.
Because if value is not changed, the `PhysicsMaterial.EVENT_UPDATE` event should not be emitted.